### ISSUE TITLE
Can i Use your toolkit with Google Maps  api ?

### ISSUE THREAD

**Issue by aboessa** [ffwdme/ffwdme.js#31]
Can i Use your toolkit with Google Maps  api ?

**Comment by fabrik42** [comment]
Hi!
Yes, you could write an adapter for the Google Maps API, to use it for routing and create a map widget for the UI. However, currently there is no support for Google Maps, because afaik the Terms of Service of Google Maps prohibit the usage for real time navigation. 

I hope this helps!

**Comment by aboessa** [comment]
I understand that the routing in only the service you support for google
and no nteractive GPS driving directions .

Or I can do all the services by FFWDME


On June 6, 2017 at 8:41:53 PM, Christian Bäuerlein (notifications@github.com)
wrote:

Hi!
Yes, you could write an adapter for the Google Maps API, to use it for
routing and create a map widget for the UI. However, currently there is no
support for Google Maps, because afaik the Terms of Service of Google Maps
prohibit the usage for real time navigation.

I hope this helps!

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/ffwdme/ffwdme.js/issues/31#issuecomment-306579544>, or mute
the thread

**Comment by fabrik42** [comment]
Sorry, but I did not understand this last message. Could you rephrase it please?

**Comment by aboessa** [comment]
how to use it by google , Can you give me an example ?

2017-06-30 17:52 GMT+02:00 Christian Bäuerlein <notifications@github.com>:

**Comment by fabrik42** [comment]
Sorry I can not help you with this, as this would be against the Terms of Service  of Google Maps.

**Comment by JoceM** [comment]
Hello, The New Google Maps T&C (eff June 11th 2018) do not fully prohibit real time navigations anymore. Are you planning to add gMap support to your great code ?
Thanks

**Comment by fabrik42** [comment]
Interesting! Do you have a link to this?

**Comment by JoceM** [comment]
https://cloud.google.com/maps-platform/terms/

Look at 3.2.4 (c). Terms have changed. Turn by Turn real time navigation is
allowed except if it is 'substantially similar to the functionality
provided by the Google Maps for Android mobile app'. To be honest, those
terms are pretty vague and subject to interpretation. The important point,
IMO, is that they went from a 'no' to a 'yes but'...


2018-05-15 9:33 GMT+02:00 Christian Bäuerlein <notifications@github.com>:

**Comment by fabrik42** [comment]
Cool! Right now I have no plans to add support for Google Maps myself, but I am happy to assist/review PRs. :)
